### PR TITLE
[script.skinshortcuts] v2.0.0@matrix

### PR DESCRIPTION
**Pull request by anxdpanic**
### Description

- restructured (no change to API)
- added copyright and spdx license headers
- housekeeping
    - cleaned up whitespace
    - unused variables
    - unused imports
    - renamed shadowed builtins
    - applied PEP8
    - linted with pylint, isort
    - updated/removed po file comments and metadata
- moved shared/common constants to their own module
- refactored json rpc requests and moved to a common module
- refactored logging and moved to a common module
- refactored hash generation and moved to a common module
- refactored file handling and moved to a common module
- removed any unnecessary string encoding/decoding
- removed duplicate code
- assorted minor fixes
- updated settings.xml to Kodi's new settings format
- added to Weblate for translations
- added ::ENABLE:: to enable an add-on, used like ::INSTALL::
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.skinshortcuts/addon.xml [hunk 1]
```diff
@@ -1,41 +1,64 @@
...
-		<license>GPL-2.0-only</license>
-		<forum>https://forum.kodi.tv/showthread.php?tid=178294</forum>
-		<source>https://github.com/mikesilvo164/script.skinshortcuts</source>
-		<assets>
-			<icon>icon.png</icon>
-			<fanart>fanart.jpg</fanart>
-		</assets>
-		<news>Kodi v19 Matrix + Python 3 compatibility</news>
-	</extension>
+<addon id="script.skinshortcuts" name="Skin Shortcuts" version="2.0.0" provider-name="BigNoid, marcelveldt, SiLVO">
```
Comment (enen92) [inline script.skinshortcuts/addon.xml:2]: I guess you need to be an author too?

File: script.skinshortcuts/addon.xml [hunk 2]
```diff
@@ -1,41 +1,64 @@
...
+- removed duplicate code
+- assorted minor fixes
+- updated settings.xml to Kodi's new settings format
+- added to Weblate for translations
+- added ::ENABLE:: to enable an add-on, used like ::INSTALL::
+        </news>
+        <platform>all</platform>
+        <license>GPL-2.0-only</license>
+        <forum>https://forum.kodi.tv/showthread.php?tid=178294</forum>
+        <source>https://github.com/mikesilvo164/script.skinshortcuts</source>
```
Comment (enen92) [inline script.skinshortcuts/addon.xml:36]: Still valid?

File: script.skinshortcuts/addon.xml (hunk 1 above)
Comment (anxdpanic) [reply script.skinshortcuts/addon.xml:2]: This is still under SiLVO's ownership/authorship, though I've taken over regular maintenance at the moment.

File: script.skinshortcuts/addon.xml (hunk 2 above)
Comment (anxdpanic) [reply script.skinshortcuts/addon.xml:36]: Still valid, will be maintained at the same repo

**Review by enen92 — APPROVED** [review]

**Pull request merged**